src/Datadog.Trace.ClrProfiler.Managed/Integrations/RedisHelper.cs

```diff
@@ -36,10 +36,12 @@ internal static Scope CreateScope(Tracer tracer, string integrationName, string
                     command = rawCommand;
                 }
 
+                scope = tracer.StartActive(command ?? OperationName, serviceName: serviceName);
+
                 var span = scope.Span;
-                span.Type = SpanTypes.Redis;
-                span.ResourceName = command;
-                span.SetTag(Tags.RedisRawCommand, rawCommand);
+                span.SetTag(Tags.InstrumentationName, integrationName);
+                span.SetTag(Tags.DbType, SpanTypes.Redis);
+                span.SetTag(Tags.DbStatement, rawCommand.Length > 1024 ? rawCommand.Substring(0, 1024) : rawCommand);
```

Comment:
nit: careful hand compiled
```suggestion
                span.SetTag(Tags.DbStatement, rawCommand.Substring(0, Math.Min(rawCommand.Length, 1024)));
```

Reply:
Since this pattern repeats a lot perhaps having a truncate helper seems a good idea. Also, the value 1024 happens in many places of the code it will be good to coalesce but let's do that as a separate PR.